R/texreg.R

```diff
@@ -6,13 +6,13 @@
 # screenreg function
```

Comment:
Have you looked at the `plotreg` function as well? Don't do it if it's too complicated, but it would obviously be better if the same set of arguments was present in all four functions. That said, I don't remember if `plotreg` was actually structured in a similar way as the other three functions.